public/app/features/plugins/admin/components/Changelog.tsx

```diff
@@ -0,0 +1,46 @@
+import { css, cx } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+import { CatalogPlugin } from '../types';
+
+interface Props {
+  plugin: CatalogPlugin;
+}
+
+export const Changelog = ({ plugin }: Props) => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <div
+      dangerouslySetInnerHTML={{ __html: plugin.details?.changelog ?? 'No changelog was found' }}
```

Comment:
I know that most likely the html coming here is from grafana-data `renderMarkdown` which already sanitizes the html but this general component is accepting a whole plugin as a prop which could have someone using this component and passing a plugin with a potentially unsanitized changelog.

I suggest you change this component to directly accept a prop called "sanitizedHTML" (you don't really need all the plugin anyway) so it is clear the developer using this component should be passing a sanitized value